zuchini-runner/src/main/java/org/zuchini/runner/Converters.java

```diff
@@ -116,29 +166,31 @@ public E convert(@Nullable String argument) {
                 return Enum.valueOf(enumClass, argument);
             }
         }
+
     }
 
     private static final Annotation[] EMPTY_ANNOTATIONS = new Annotation[0];
+
     private static final Map<Class<?>, Converter<?>> DEFAULT_CONVERTERS;
 
     static {
         Map<Class<?>, Converter<?>> map = new HashMap<>(32);
-        map.put(Byte.TYPE, DefaultConverters.BYTE);
-        map.put(Byte.class, DefaultConverters.BYTE);
-        map.put(Short.TYPE, DefaultConverters.SHORT);
-        map.put(Short.class, DefaultConverters.SHORT);
-        map.put(Integer.TYPE, DefaultConverters.INT);
-        map.put(Integer.class, DefaultConverters.INT);
-        map.put(Long.TYPE, DefaultConverters.LONG);
-        map.put(Long.class, DefaultConverters.LONG);
-        map.put(Float.TYPE, DefaultConverters.FLOAT);
-        map.put(Float.class, DefaultConverters.FLOAT);
-        map.put(Double.TYPE, DefaultConverters.DOUBLE);
-        map.put(Double.class, DefaultConverters.DOUBLE);
-        map.put(Character.TYPE, DefaultConverters.CHARACTER);
-        map.put(Character.class, DefaultConverters.CHARACTER);
-        map.put(Boolean.TYPE, DefaultConverters.BOOLEAN);
-        map.put(Boolean.class, DefaultConverters.BOOLEAN);
+        map.put(Byte.TYPE, DefaultConverters.BYTE_PRIMITIVE);
```

Comment:
Is`Byte.TYPE` the same as `byte.class`?

Reply:
Yes, did not realize that `byte.class` is actually valid syntax, why would they need those constants otherwise?

Reply:
Wouldn't it help in case someone calls e.g. `DefaultConverters.BOOLEAN.convert("true")` directly?